assets/css/tasks/_article-tasks.css

```diff
@@ -0,0 +1,114 @@
+/* SINGLE ARTICLE PAGE */
+.container {
+    font-family: 'Lato', sans-serif;
+}
+
+.post-page__img { 
+    width: 100%;
+    max-height: 250px;
+    object-fit: contain;
+    overflow: hidden;
+    object-fit: cover;
+    padding: 0px 250px 0px 250px;
+    margin-top: 50px;
+}
+
+.post-page__header {
+    width: 100%;
+    padding-bottom: 100px;
+}
+
+.u-margin--medium {
+    margin: 100px 350px 50px 350px;
+}
+
+.post-description__title {
+    float: left;
+    margin-bottom: 30px;
+}
+
+.post-description__date {
+    float: right;
+}
+
+.post-end {
+    margin: 100px 300px 50px 300px;
+}
+
+.post-page-end__button--previous,
+.post-page__author-info,
+.post-page-end__button--next {
+    float: left;
+    width: 33%;
+    /* width; all three elements at the end of the page should share the available width between them, so one element should take up 1/3 of the entire available width*/
```

Comment:
That is right. It should be somehow 30% of total width.
So you can delete this commented text. I know, that comment was from the begining, but it is better practice to rid of from useless comments. Somethimes, comments are needed but here you did your job and comment doesn't breeng more usefull information anymore.